Nodejs/Product/PressAnyKey/PressAnyKey.cs

```diff
@@ -19,22 +20,35 @@ private static int Main(string[] args)
             }
 
             Console.Title = args[2];
-            var psi = new ProcessStartInfo(args[2], string.Join(" ", args.Skip(3).Select(arg => ProcessOutput.QuoteSingleArgument(arg))));
-            psi.UseShellExecute = false;
+            var psi = new ProcessStartInfo(args[2], string.Join(" ", args.Skip(3).Select(arg => ProcessOutput.QuoteSingleArgument(arg))))
+            {
+                UseShellExecute = false
+            };
 
-            var proc = Process.Start(psi);
-            File.WriteAllText(args[1], proc.Id.ToString());
-            proc.WaitForExit();
+            var exitCode = 0;
+            try
+            {
+                var proc = Process.Start(psi);
+                File.WriteAllText(args[1], proc.Id.ToString());
+                proc.WaitForExit();
+                exitCode = proc.ExitCode;
+            }
+            catch (Win32Exception exc)
+            {
+                Console.WriteLine($"Failed to start process: '{exc.Message}'.");
+                Console.WriteLine("Probable cause is the Nodejs exe is corrupt, please re-install.");
```

Comment:
If we're displaying this to the user, we probably have to be more careful about names.  I think they use "Node.js".